source/_components/folder_watcher.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "folder watcher"
```

Comment:
This should be cased properly as `Folder Watcher`

Reply:
ok